stdlib/public/Python/Python.swift

```diff
@@ -1253,6 +1253,21 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : IteratorProtocol, Sequence {
+
+  public func makeIterator() -> PythonObject {
+    return Python.iter(self)
+  }
+
+  public func next() -> PythonObject? {
+     guard let result: OwnedPyObjectPointer = PyIter_Next(self.borrowedPyObject) else {
+      try! throwPythonErrorIfPresent()
```

Comment:
Indent by 2.

Reply:
Not sure I’m missing anything, but these lines are indented by 3 instead of 2. The indentation convention in this repo is by 2 spaces.